src/components/CommentList/index.tsx

```diff
@@ -2,20 +2,19 @@ import * as React from 'react';
 import { connect } from 'react-redux';
 
 import { ApplicationState } from '../../reducers';
-import {
-  CommentKeyParams,
-  CommentInfo,
-  createCommentKey,
-} from '../../reducers/comments';
+import { CommentInfo, selectCommentInfo } from '../../reducers/comments';
 import Comment from '../Comment';
 import styles from './styles.module.scss';
 
 export type ChildrenArgValue = JSX.Element;
 
-export type PublicProps = CommentKeyParams & {
```

Comment:
I've decided that this was not a good idea to begin with. It's not the right place to keep them in sync. When editing `CommentKeyParams`, I had removed `versionId` but `CommentList` still used the `versionId` prop for other things. It created some confusion because these props are not used entirely for building keys; thus, it's not helpful to bake `CommentKeyParams` into the prop types.